resources/settings.xml

```diff
@@ -6,13 +6,15 @@
     <setting label="32002" type="text" id="elementum.jackett.api_key" />
     <setting label="32003" type="text" id="elementum.jackett.settings_validated" default="" enable="false" />
     <setting label="32004" type="action" action="RunScript(script.elementum.jackett, validate_settings)" option="close" />
+    <setting label="32007" type="bool" id="elementum.jackett.search_by_imdb_key"  default="false" />
```

Comment:
Need old behaviour as default for backwards compatibility.

```suggestion
    <setting label="32007" type="bool" id="elementum.jackett.search_by_imdb_key"  default="true" />
```

Reply:
I will. But in my experience search by name is much more reliable compare to by_imdb. 
So imho default search by name is better.

Reply:
> Need old behaviour as default for backwards compatibility.

I will. But in my experience search by name is much more reliable compare to by_imdb.
So imho default search by name is better.

Reply:
done

Reply:
> I will. But in my experience search by name is much more reliable compare to by_imdb.
> So imho default search by name is better.

Hmm, this is fair. Kinda wonder if it makes sense to just raise an alert, or we do just revert it.